source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -237,6 +237,22 @@
             return valuesByKey;
         });
 
+        var commentsByKeyCollection = _.map(data.Califications, function (calification) {
+            var commentsByKey = {
+                commentRow: {
+                    calificationId: calification.Id,
+                    evaluatorEmployee: calification.EvaluatorEmployee,
+                    owner: calification.Owner
+                }
+            };
+            if (calification.Califications)
+                for (var calif in calification.Califications) {
+                    var cal = calification.Califications[calif];
+                    commentsByKey[cal.Key] = cal.Comment;
+                }
+            return commentsByKey
```

Comment:
But if I am not wrong, the result of it will be something link:

```
{
    commentRow: {
        calificationId: 454,
        evaluatorEmployee: "abriscioli",
        owner: "amoschini"
    },
    responsability: "Le puse un uno porque Andres siempre llega tarde y nunca hace nada",
    personalPresentation: "Le puse un uno porque parece que nunca se baña y apesta",
    . . .
}
```

Is it fine? I am wrong?